std/typecons.d

```diff
@@ -3452,7 +3452,7 @@ Params:
         return isNull ? false : rhs == _value;
     }
 
-@safe unittest
+@system unittest
```

Comment:
This should not be necessary. And in fact, there are other places it's `@safe`. Something is wrong here, it appears the compiler isn't properly inferring safety.